config/develop/tower-dev-certificate.yaml

```diff
@@ -0,0 +1,11 @@
+template_path: remote/acm-certificate.yaml
+stack_name: tower-dev-certificate
+parameters:
+  Department: IBC
+  Project: Infrastructure
+  OwnerEmail: nextflow-admins@sagebase.org
+  DnsDomainName: sagebionetworks.org
+  DnsSubDomainName: tower-dev
+hooks:
+  before_launch:
+    - !cmd "wget {{stack_group_config.aws_infra_templates_root_url}}/v0.3.0/templates/acm-certificate.yaml -O templates/remote/acm-certificate.yaml"
```

Comment:
hmm, i'm not a fan of this template anymore since it's setting Dept, project, and OwnerEmail tags.  would prefer to see `stack_tags: ..` here instead.  Maybe we should create a new file in `Sage-Bionetworks/aws-infra/ACM/certificate.yaml` ?

Reply:
Sorry -- I am overwhelmed with other work. I will happily use an updated template in aws-infra but this is the available one and that it's not using stack_tags does not inhibit its functionality.